metadata-ingestion/tests/unit/urns/test_urn.py

```diff
@@ -101,6 +110,11 @@ def test_urn_type_dispatch_2() -> None:
     with pytest.raises(InvalidUrnError, match="Passed an urn of type dataJob"):
         CorpUserUrn.from_string(urn)
 
+    with pytest.raises(
+        InvalidUrnError, match="Expecting a CorpUserUrn but got.*dataJob.*"
```

Comment:
cosmetic change suggestion

```suggestion
        InvalidUrnError, match="Expecting a CorpUserUrn but got .*dataJob.*"
```